### PR TITLE
Put 21 at 30% to 22 on production

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: tests/integration/features/stable.feature [hunk 1]
```diff
@@ -761,6 +761,27 @@ Feature: Testing the update scenario of stable releases
...
+  
+  Scenario: Updating the Nextcloud 21.0.0 to 22.0.0 on the stable channel
+    Given There is a release with channel "stable"
+    And The received version is "21.0.0.7"
+    And The received PHP version is "7.3.0"
+    And the installation mtime is "11"
+    When The request is sent
+    Then The response is non-empty
+    And Update to version "22.0.0.11" is available
+    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-22.0.0.zip"
```
Comment (AndyScherzinger) [inline tests/integration/features/stable.feature:773]: shouldn't it then also be `22.0.0.11` or change line `772`, no`?

**Review by AndyScherzinger — APPROVED** [review]
All good, just a minor question regarding the test case value(s)

File: tests/integration/features/stable.feature (hunk 1 above)
Comment (skjnldsv) [reply tests/integration/features/stable.feature:773]: Maybe it should even be 22.1.0 ?
@blizzz ?

Comment (skjnldsv) [reply tests/integration/features/stable.feature:773]: > shouldn't it then also be 22.0.0.11 or change line 772, no`?

We don't use the 4th digit for the filenames :)

**Comment by skjnldsv** [comment]
Closing for circles

**Pull request closed without merging**